src/platform/packages/shared/kbn-scout/src/playwright/runner/flags.test.ts

```diff
@@ -72,6 +72,7 @@ describe('parseTestFlags', () => {
     expect(result).toEqual({
       mode: 'serverless=oblt',
       configPath: '/path/to/config',
+      testTarget: 'local',
```

Comment:
By default, if we run the Playwright tests _without_ `project` flag, my understanding is that `local` will be picked up by default, by looking at this test (and by running the tests locally).

I don't quite get why we say `Running test only requires passing project argument:` in the PR description then, if the `project` flag isn't strictly required.

Reply:
afaik it is required to pass `--project` flag, otherwise dependency project (that runs global setup hook) won't be triggered. I will post an update here

Reply:
>By default, if we run the Playwright tests without project flag, my understanding is that local will be picked up by default, by looking at this test (and by running the tests locally).

Correct, I double checked it.

Reply:
however, we want our orchestrator to be predictable as it is not only for Playwright, but for future API runner as well. We agreed with @dolaru that `testTarget` is how Scout defines the environment, later we match it with specific runner (for Playwright it is a project flag)